src/data/nat/interval.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2021 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import data.finset.interval
+
+/-!
+# Finite intervals of naturals
+
+This file proves that `ℕ` is a `locally_finite_order` and calculates the cardinality of its
+intervals as finsets and fintypes.
+
+## TODO
+
+Add all the `finset.Ico` stuff.
```

Comment:
If you could expand slightly on this (even just a sentence or two) it might be worth it.

Reply:
I could, but this will literally be fixed by the second or third next PR in this series.